### ISSUE TITLE
cobol unreachable code

### ISSUE THREAD

**Issue by Ldspout** [IBM/zopeneditor-about#7]
Hi,

Ho do I proceed to see it in VS Code ?
thank you,
L.

**Comment by phaumer** [comment]
Hello @Ldspout. Thanks for your interest in our extension. Unreachable Code right now will be shown as an error in the Problems view. We have an open enhancement request open to turn it into a warning as well as make it configurable (i.e. allow users to switch it off).

Open it via the menu View > Problems. There is also an icon in the status bar showing you the counts. You can click on that as well. Then click on any entry in the list to jump to the offending line. 

![image](https://user-images.githubusercontent.com/4391934/64964988-ffacaa80-d850-11e9-89b1-70a732753ec6.png)

**Comment by Ldspout** [comment]
Hi, 

thank you for your quick feeback, 
I've tested on a -very old- cobol source. Does it work with cobol free format ? 
Does it work with big source bases ? 
for an example, 

GO TO TT-CLAUSES. 

ESSAI-CODE-MORT. 
MOVE CONT-TX-APPEL TO CONT-TYPE-CONTRAT. 

TT-CLAUSES. 
MOVE 0 TO I. 
MOVE ZEROES TO AIG-CLAUSES. 

doesn't print any unreachable error on the "ESSAI-CODE-MORT" 

Are there extensions who might cause this ? 
thank you, 

all the best, 
Loïc 

----- Mail original -----

**Comment by phaumer** [comment]
Hello. Not sure what you mean be free format. We support [IBM Enterprise COBOL](https://www.ibm.com/support/knowledgecenter/en/SS6SG3_6.2.0/welcome.html) only. But, when I paste in your code I do see Unreachable Code errors. If you so not see any errors at all, such as syntax, perhaps the extension is not correctly installed. Did you check the section on configuring Java? 

![image](https://user-images.githubusercontent.com/4391934/65072335-8642b400-d945-11e9-93bc-33a787fbdd7b.png)

**Comment by phaumer** [comment]
We found a case where there could be a problem with showing this error. We are investigating..

**Comment by Morridt** [comment]
Would this also be the case with showing any errors?
I have the syntax highlighting; outlines; references look-ups; hovers; external copybook includes.  

But no systax etc errors. (ie nothing in problems, or Output view).

User environment variables
JAVA_HOME in  is pointing to JDK  1.8.0_144 (64-bit)
PATH includes %JAVA_HOME%\bin

System Environment variables are being picked up first and pointing to 1.8.0_73 ...assumed JRE.

```
C:\Users\n089338\jdk\bin>java -version
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)

C:\Users\n089338\jdk\bin>cd ..

C:\Users\n089338\jdk>java -version
java version "1.8.0_73"
Java(TM) SE Runtime Environment (build 1.8.0_73-b02)
Java HotSpot(TM) Client VM (build 25.73-b02, mixed mode)

C:\Users\n089338\jdk>where java
C:\ProgramData\Oracle\Java\javapath\java.exe
C:\Users\n089338\jdk\bin\java.exe
```

**Comment by phaumer** [comment]
@Morridt do you have a small sample program you can share here where this can be reproduced?

**Comment by Morridt** [comment]
yep..Stripped this one back to bare bones checking along the way.  And as soon as I got rid of the EJECT statements things started working.
regards.... Darren

**Comment by phaumer** [comment]
This issue was fixed in version 0.4.1 now available on the VS Code Marketplace.
